### PR TITLE
(FB: 153159) - Sort report columns by name

### PR DESCRIPTION
**Pull request by benrudolph**
Sort the product in the Inventory Report by the product name instead of
by the id.

http://manage.dimagi.com/default.asp?153159#862494

@twymer i think this is your domain

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: corehq/apps/reports/commtrack/standard.py [hunk 1]
```diff
@@ -272,7 +268,7 @@ def rows(self):
             row_dict = dict(self.product_dict, **dict(loc_data))
             yield [loc_name] + [
                 v if v is not None else _('No data')
-                for k, v in sorted(row_dict.items(), key=lambda(k, v): k)]
+                for k, v in sorted(row_dict.items(), key=lambda(k, v): self.product_names[k])]
```
Comment (czue) [inline corehq/apps/reports/commtrack/standard.py]: can the sort key not just be `v`?

Comment (benrudolph) [inline corehq/apps/reports/commtrack/standard.py]: in this case `row_dict` is a mapping between product_id to the value of that product, like 50, 60, etc, so we'd be sorting by product value not product name

Comment (czue) [inline corehq/apps/reports/commtrack/standard.py]: makes sense - sorry was reading too quickly

**Comment by twymer** [comment]
:+1:

**Pull request merged**